beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -35,11 +29,6 @@ func newLightClientOptimisticUpdateFromBeaconState(
 	block interfaces.ReadOnlySignedBeaconBlock,
 	attestedState state.BeaconState,
 	attestedBlock interfaces.ReadOnlySignedBeaconBlock,
-) (*structs.LightClientOptimisticUpdate, error) {
-	result, err := lightclient.NewLightClientOptimisticUpdateFromBeaconState(ctx, currentSlot, state, block, attestedState, attestedBlock)
-	if err != nil {
-		return nil, err
-	}
-
-	return structs.LightClientOptimisticUpdateFromConsensus(result)
+) (interfaces.LightClientOptimisticUpdate, error) {
+	return lightclient.NewLightClientOptimisticUpdateFromBeaconState(ctx, currentSlot, state, block, attestedState, attestedBlock)
```

Comment:
We can call this function directly from handlers, not need to go through the helper function that doesn't do anything.